src/main/java/utils/DriverFactory.java

```diff
@@ -0,0 +1,64 @@
+package utils;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.testng.annotations.Parameters;
+
+import pages7daySuperMarket.LoginPage;
+
+public class DriverFactory {
+	
+
+	
+	public static WebDriver driverInitialization(String browserName ) {
+		
+		if(browserName.equals("firefox"))
+		{
+			return getFireFoxDriver();
+		}
+		
+		else if(browserName.equals("chrome"))
+		{
+			return getChromeDriver();
+		}
+		
+		else
+		{
+			return getChromeDriver();
+		}
+	}
+	
+	private static WebDriver getChromeDriver() {
+		System.setProperty("webdriver.chrome.driver",
+				"C:\\Program Files\\webdrivers\\chromedriver_win32\\chromedriver.exe");
```

Comment:
consider these items added in a property file/Yaml file  and read it from here
improvements